db/db_options_test.cc

```diff
@@ -117,6 +117,28 @@ TEST_F(DBOptionsTest, GetLatestCFOptions) {
             GetMutableCFOptionsMap(dbfull()->GetOptions(handles_[1])));
 }
 
+TEST_F(DBOptionsTest, SetBytesPerSync) {
```

Comment:
Can you also verify after setting the option, flush/compaction/wal is using the new bytes_per_sync value? You can inject `TEST_SYNC_POINT_CALLBACK` in `WritableFileWriter::RangeSync()` to count how many times it is called after triggering flush/compaction or writing some data into db with `write_options.disable_wal = false && write_options.sync = false`.

Reply:
Nice test!